### ISSUE TITLE
PSModulePath default on Linux is under powershell and InstallHelper hard-codes Powershell

### ISSUE THREAD

**Issue by ChrisAtRelativity** [PowerShell/PSResourceGet#218]
### Impact
By default, installing a module with Install-PSResource fails on Linux.

### Reproduce

* Install Powershell 7.0.3 on Ubuntu 18.04 as root
* Use PS Get 2.2 to Install-Module modules from PS Gallery.
 * Modules are installed to `/root/.local/share/powershell/Modules`
 * PSModulePath is `/root/.local/share/powershell/Modules:/usr/local/share/powershell/Modules:/opt/microsoft/powershell/7/Modules`
* Install PS Get 3.0 beta 8
* Register-PSResourceRepository a new repo (successful).
* Install-PSResource a new module fails. Error is `Install-PSResource: Could not find a part of the path '/root/.local/share/Powershell/Modules'.`

InstallHelper [hard codes](https://github.com/PowerShell/PowerShellGet/blob/master/src/InstallHelper.cs#L85) an uppercase Powershell directory.

PSModulePath for PS7 is [documented](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_psmodulepath?view=powershell-7) as a lowercase powershell directory.
```
On non-Windows systems, the location of the user-specific CurrentUser scope is the $HOME/.local/share/powershell/Modules folder. The location of the AllUsers scope is /usr/local/share/powershell/Modules.
```

**Comment by alerickson** [comment]
@ChrisAtRelativity thanks for opening this issue!  I'm actually working on this issue right now and it'll be resolved with the next release

**Comment by ChrisAtRelativity** [comment]
@alerickson I think there are some more case-sensitivity issues than just this one, too, with the case-sensitive module name itself.

Next Reproduction:
* Create Symbolic link `ln -s /root/.local/share/powershell /root/.local/share/Powershell`
* Attempt to Install-PsResource (fails)
* Error: `Install-PSResource: Could not find a part of the path '/tmp/b12b49d2-5c8a-4133-a46d-18f18d2bbbe4/ModuleName/1.2.3/modulename.1.2.3.nupkg'.`
```
ls /tmp/b12b49d2-5c8a-4133-a46d-18f18d2bbbe4 
modulename
```

**Comment by alerickson** [comment]
This is resolved in [3.0.0-beta10](https://www.powershellgallery.com/packages/PowerShellGet/3.0.0-beta10)
